ext/bcmath/bcmath.c

```diff
@@ -1185,6 +1186,11 @@ static zend_result bcmath_number_do_operation(uint8_t opcode, zval *ret_val, zva
 		goto fail;
 	}
 
+	if (UNEXPECTED(CHECK_SCALE_OVERFLOW(n1_full_scale) || CHECK_SCALE_OVERFLOW(n2_full_scale))) {
+		zend_value_error("Number has too large scale");
```

Comment:
The wording of this error is very clunky.

Surely we can use a more standard phrasing like `scale must be between 0 and %d` with INT_MAX provided for `%d` ?

Reply:
Thanks, I'll fix later!

Reply:
@Girgias 
done! https://github.com/php/php-src/pull/15741/commits/675af48663d21c4a208b9c4ca28c8f74198b4fd5

Reply:
Unrelated tests are failing